TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterStatus.java

```diff
@@ -0,0 +1,298 @@
+                }
+            }
+        }
+
+        // Media
+        MediaEntity[] medias = status.getMediaEntities();
+        if (medias != null && medias.length != 0) {
+            int start = medias[0].getStart();
+            int end = medias[0].getEnd();
+            if (end == status.getText().length()) {
+                start += offset;
+                end += offset;
+            }
+            if (ssb.length() > start && ssb.length() >= end) {
+                if (PrefAppearance.isShowThumbnail()) {
+                    ssb.replace(start, end, "");
+                } else {
+                    String displayURL = medias[0].getDisplayURL();
+                    ssb.replace(start, end, displayURL);
+                    ssb.setSpan(new ForegroundColorSpan(color), start, start + displayURL.length(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+                }
+            }
+        }
+
+        // Convert to string
+        String htmlText = HtmlCompat.toHtml(ssb, TO_HTML_PARAGRAPH_LINES_INDIVIDUAL);
+        String match = "<p dir=\"ltr\" style=\"margin-top:0; margin-bottom:0;\">";
+        htmlText = htmlText.replaceFirst("(?s)" + match + "(?!.*?" + match + ")", "");
+        htmlText = htmlText.replaceFirst("(?s)" + "</p>" + "(?!.*?" + "</p>" + ")", "");
+        htmlText = htmlText.replaceAll("\n", "");
+
+        return htmlText;
+    }
+
+    private static String createRelativeTime(Status status) {
+        // Convert
+        Date date = status.getCreatedAt();
+        String text = new TimeSpanConverter().toTimeSpanString(date);
+
+        // Change text
+        text = text.replace("前", ""); //「〜分前」→「〜分」
+        text = text.replace("月", "/"); //「10月29日」→「10/29」
+        text = text.replace("日", "");
+
+        return text;
+    }
+
+    private static String createAbsoluteTime(Status status) {
+        // Convert
+        Date date = status.getCreatedAt();
+        return new SimpleDateFormat("y/M/d HH:mm", Locale.getDefault()).format(date);
+    }
+
+    private static String createVia(Status status) {
+        // Convert
+        String[] texts = status.getSource().split("[<>]");
+        if (texts.length > 2) {
+            return "via " + texts[2];
+        } else {
+            return "via Unknown Client";
+        }
+    }
+
+    private static ArrayList<String> createImageUrls(Status status) {
+        // Init
+        ArrayList<String> imageUrls = new ArrayList<>();
+        Status source = status.isRetweet() ? status.getRetweetedStatus() : status;
+
+        // Check media
+        if (source.getMediaEntities().length > 0) {
+            // Twitter media
+            MediaEntity[] medias = source.getMediaEntities();
+            for (int i = 0; i < medias.length; i++) {
+                imageUrls.add(medias[i].getMediaURLHttps());
+            }
+        } else {
+            // Third party media
+            for (URLEntity url : source.getURLEntities()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS135)